### PR TITLE
feat: Expose apis for list get and update customer operations

### PR DESCRIPTION
**Pull request by jredl-va**
@vendasta/external-apis
@vendasta/data-busters 
@DewaldoDev  (if you have time to review) 
@hchow-va (if you have time to review)

Expose apis for list, get and update. This extends the existing create customer api that exists. Note, that I've removed some of the "business app" verbiage as we want to ensure its understood that these apis can be used by marketplace / smb products.

### CODE REVIEW

**Comment by lwang-va** [comment]
I noticed that the customers APIs are added as part of Business REST APIs. Does it mean that the customers would be added into the centralized data base of Business App and then synced to individual products like Customer Voice ?

File: openapi/business/business.yaml [hunk 1]
```diff
@@ -208,6 +208,183 @@ paths:
...
+                type: object
+                properties:
+                  data:
+                    $ref: '#/components/schemas/customers'
+                  links:
+                    type: object
+                    properties:
+                      self:
+                        type: string
+      operationId: get-customers-id
```
Comment (lwang-va) [inline openapi/business/business.yaml:300]: I guess that this operation is actually to get a customer by id, is it correct ?

File: openapi/business/business.yaml [hunk 2]
```diff
@@ -208,6 +208,189 @@ paths:
...
+        - schema:
+            type: string
+          in: header
+          name: Authorization
+          description: A Bearer access token to identify the user the app is acting on behalf of. See the Authorization guide for details.
+          required: true
+        - schema:
+            type: string
+          in: query
+          name: 'filter[businesses.id]'
```
Comment (DewaldoDev) [inline openapi/business/business.yaml:254]: I think we've been opting for the term `businessLocation` when referencing account groups.

**Review by DewaldoDev — APPROVED** [review]

File: openapi/business/business.yaml (hunk 2 above)
Comment (richard-rance) [reply openapi/business/business.yaml:254]: Yes. `filter[businessLocation.id]` is what the majority of endpoints use.

**Pull request merged**